### ISSUE TITLE
Aggregate tables

### ISSUE THREAD

**Issue by DavidGOrtega** [DavidGOrtega/dvc-action#11]
JSON metrics if they are objects can be actually painted as a table.

The idea is to create a table with all the metrics. So output would be this:

<details>
<summary>metrics/train.json</summary>

|id|batch_size|num_steps|learning_rate|took|
|----|----|----|----|----|
|current|128|2300|0.05|0.004099244594573974| 
|[1234567](https://gist.github.com/DavidGOrtega/705d7c1327c3a20bc069e3f1d09d4548)|128|2800|0.03|0.004099244594573974| 
|[1234567](https://gist.github.com/DavidGOrtega/705d7c1327c3a20bc069e3f1d09d4548)|128|3000|0.05|0.004099244594573974| 
|[1234567](https://gist.github.com/DavidGOrtega/705d7c1327c3a20bc069e3f1d09d4548)|128|5000|0.05|0.004099244594573974|  
|[1234567](https://gist.github.com/DavidGOrtega/705d7c1327c3a20bc069e3f1d09d4548)|128|2800|0.03|0.004099244594573974| 
|[1234567](https://gist.github.com/DavidGOrtega/705d7c1327c3a20bc069e3f1d09d4548)|128|3000|0.05|0.004099244594573974| 
</details>

<details>
<summary>metrics/eval.json</summary>

|id|accuracy|
|----|----|
|current|0.9106|  
|[1234567](https://gist.github.com/DavidGOrtega/705d7c1327c3a20bc069e3f1d09d4548)|0.9206|  
|[1234567](https://gist.github.com/DavidGOrtega/705d7c1327c3a20bc069e3f1d09d4548)|0.9306|  
|[1234567](https://gist.github.com/DavidGOrtega/705d7c1327c3a20bc069e3f1d09d4548)|0.9406|  
|[1234567](https://gist.github.com/DavidGOrtega/705d7c1327c3a20bc069e3f1d09d4548)|0.9406|  
|[1234567](https://gist.github.com/DavidGOrtega/705d7c1327c3a20bc069e3f1d09d4548)|0.9406|  
</details>

**Comment by DavidGOrtega** [comment]
|id|batch_size|num_steps|learning_rate|took|accuracy|
|----|----|----|----|----|----|
|current|128|2300|0.05|0.004099244594573974| 0.9106|  
|[1234567](https://gist.github.com/DavidGOrtega/705d7c1327c3a20bc069e3f1d09d4548)|128|2800|0.03|0.004099244594573974| 0.9106| 
|[1234567](https://gist.github.com/DavidGOrtega/705d7c1327c3a20bc069e3f1d09d4548)|128|3000|0.05|0.004099244594573974| 0.9106| 
|[1234567](https://gist.github.com/DavidGOrtega/705d7c1327c3a20bc069e3f1d09d4548)|128|5000|0.05|0.004099244594573974|0.9106|   
|[1234567](https://gist.github.com/DavidGOrtega/705d7c1327c3a20bc069e3f1d09d4548)|128|2800|0.03|0.004099244594573974| 0.9106| 
|[1234567](https://gist.github.com/DavidGOrtega/705d7c1327c3a20bc069e3f1d09d4548)|128|3000|0.05|0.004099244594573974| 0.9106|
